tests/WE2E/monitor_jobs.py

```diff
@@ -180,6 +179,8 @@ def setup_logging(logfile: str = "log.run_WE2E_tests", debug: bool = False) -> N
     parser.add_argument('-y', '--yaml_file', type=str,
                         help='YAML-format file specifying the information of jobs to be run; '\
                              'for an example file, see monitor_jobs.yaml', required=True)
+    parser.add_argument('--delay', type=int,
+                        help='Pause this number of seconds between calls to rocotorun', default=10)
```

Comment:
Same as above, default for `delay` should be `5`.

```suggestion
    parser.add_argument('--delay', type=int,
                        help='Pause this number of seconds between calls to rocotorun', default=5)
```